distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Split_Tokenize.enso

```diff
@@ -234,3 +348,25 @@ handle_nothing function = x-> case x of
 ## PRIVATE
    Repeat a computation n times.
 repeat_each n ~action = 0.up_to n . each _-> action
+
+## PRIVATE
+   Name a column by appending an integer to a base column name.
+default_column_namer : Text -> Integer -> Text
+default_column_namer base_name i = base_name + " " + i.to_text
+
+## PRIVATE
+   Pad or truncate a vector to be a specified length; if altered, report
+   it as a Column_Count_Mismatch warning.
+uniform_length : Integer -> Vector Any -> Problem_Builder -> Vector Any
+uniform_length target_length v problem_builder = if v.length == target_length then v else
+    problem = Column_Count_Mismatch.Error target_length v.length
+    problem_builder.report_other_warning problem
+    case v.length < target_length of
+        True -> # Pad.
+            num_paddings = target_length - v.length
+            new_v = Vector.new_builder
```

Comment:
```suggestion
            new_v = Vector.new_builder target_length
```

We know the target length up front, so we should initialize the builder with proper initial capacity, to avoid reallocations/wasted space.